### ISSUE TITLE
When set 'logger=None' in write_videofile function it goes error:  [WinError 6] The handle is invalid

### ISSUE THREAD

**Issue by shawnxiaoxiong** [Zulko/moviepy#1261]
When set 'logger=None' in write_videofile function it goes error:  [WinError 6] The handle is invalid

```
final_clip.write_videofile(clip_target_path,logger=None)
```

#### Expected Behavior
Successful

#### Actual Behavior
```
Exception ignored in: <function FFMPEG_VideoReader.__del__ at 0x0C826418>
Traceback (most recent call last):
  File "C:\Users\xiongx\AppData\Local\Programs\Python\Python38-32\lib\site-packages\moviepy\video\io\ffmpeg_reader.py", line 199, in __del__
    self.close()
  File "C:\Users\xiongx\AppData\Local\Programs\Python\Python38-32\lib\site-packages\moviepy\video\io\ffmpeg_reader.py", line 190, in close
    self.proc.terminate()
  File "C:\Users\xiongx\AppData\Local\Programs\Python\Python38-32\lib\subprocess.py", line 1442, in terminate
    _winapi.TerminateProcess(self._handle, 1)
OSError: [WinError 6] The handle is invalid
```

#### Steps to Reproduce the Problem
Just set  'logger=None' in write_videofile function

#### Specifications

Python Version: python3.8.3
Moviepy Version: 1.0.3 
Platform Name: windows 10
Platform Version: windows 10
ffmprg version:ffmpeg version 4.2.3

**Comment by raj6996** [comment]
`
`                     | 2/320 [00:00<00:17, 18.29it/s, now=None]Traceback (most recent call last):
  File "C:\Python38\lib\runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Python38\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\Python38\lib\site-packages\videogrep\silence.py", line 64, in <module>
    silence(inputfile=args.inputfile, outputfile=args.outputfile, maxclips=args.maxclips, minsilence=args.minsilence, maxsilence=args.maxsilence, padding=args.padding, test=args.test, randomize=args.randomize)
  File "C:\Python38\lib\site-packages\videogrep\silence.py", line 43, in silence
    videogrep.create_supercut_in_batches(composition, outputfile, 0)
  File "C:\Python38\lib\site-packages\videogrep\videogrep.py", line 240, in create_supercut_in_batches
    video = concatenate(clips)
  File "C:\Python38\lib\site-packages\moviepy\tools.py", line 137, in fdepr
    return f(*a, **kw)
  File "C:\Python38\lib\site-packages\moviepy\video\compositing\concatenate.py", line 75, in concatenate_videoclips
    w = max(r[0] for r in sizes)
ValueError: max() arg is an empty sequence



                                                                                                                                                                                       Exception ignored in: <function FFMPEG_VideoReader.__del__ at 0x0000022B8E9B7F70>
Traceback (most recent call last):
  File "C:\Python38\lib\site-packages\moviepy\video\io\ffmpeg_reader.py", line 199, in __del__
    self.close()
  File "C:\Python38\lib\site-packages\moviepy\video\io\ffmpeg_reader.py", line 190, in close
    self.proc.terminate()
  File "C:\Python38\lib\subprocess.py", line 1442, in terminate
    _winapi.TerminateProcess(self._handle, 1)
OSError: [WinError 6] The handle is invalid
Exception ignored in: <function FFMPEG_AudioReader.__del__ at 0x0000022B8E9BE550>
Traceback (most recent call last):
  File "C:\Python38\lib\site-packages\moviepy\audio\io\readers.py", line 254, in __del__
    self.close_proc()
  File "C:\Python38\lib\site-packages\moviepy\audio\io\readers.py", line 149, in close_proc
    self.proc.terminate()
  File "C:\Python38\lib\subprocess.py", line 1442, in terminate
    _winapi.TerminateProcess(self._handle, 1)
OSError: [WinError 6] The handle is invalid`

`
Use latest videogrep and check moviepy version 1.0.3 and Ffmpeg also latest version from official site.
you closed issue for not updating moviepy. anyway check this error and kindly fix it. thanks.

**Comment by keikoro** [comment]
@raj6996 I saw you were apparently trying to format your code, but used the wrong syntax. For full code blocks, you need to use three backticks at the start of the block, and three at the end.

**Comment by mondeja** [comment]
> When set 'logger=None' in write_videofile function it goes error: [WinError 6] The handle is invalid

This error has not been caused by setting the logger as `None`, it's an error trying to terminate a process that is ended and has been fixed in #1296.

> ValueError: max() arg is an empty sequence

This also has been fixed in the master.

So I'm closing this. Feel free to reopen it if your problems persists and by the way thank you for the reports!
